app/views/bookings/index.html.haml

```diff
@@ -13,7 +12,8 @@
             .cal
               = booking.name
               = booking.start_time.strftime('%H:%M')
-  .row
+.row
+  .div
```

Comment:
This makes a div of class "div" - is that the intention? You can just have a blank div with `%div`, I believe.